### PR TITLE
Fixes #6481 - Updates Invoke-WebRequest/ Invoke-RestMethod

### LINKED ISSUE #6481: New Tls13 value in WebSslProtocol
# Documentation Issue

After https://github.com/PowerShell/PowerShell/pull/13409 WebSslProtocol type gets new 'Tls13' value in PowerShell 7.1.

## Context of the issue

- URL to the article (Docs or GitHub URL):
- List of articles that need to be updated:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.powershell.commands.websslprotocol?view=powershellsdk-7.0.0
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-webrequest?view=powershell-7
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-restmethod?view=powershell-7

- Issue affects the following content (check all that apply):

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

- Is the issue specific to a platform (Y/N - Win/macOS/Linux):

### PR DESCRIPTION
**Pull request by chasewilson**
# PR Summary

New Value available in Invoke-RestMethod and Invoke-WebRequest SslProtocol

## PR Context

Fixes #6481 
Fixes [AB#1767800](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1767800)

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

**Review by rjmholt — APPROVED** [review]
/cc @iSazonov

**Comment by iSazonov** [comment]
I'd add a notice in the docs that TLS 1.3 support depends on underlying OS support and currently OS-s have limited TLS 1.3 support. Otherwise users will be confused - PowerShell has TLS 1.3 support but it does not work (even on latest Windows 10).

**Comment by chasewilson** [comment]
#sign-off

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md [hunk 1]
```diff
@@ -982,15 +982,17 @@ using flag notation or combining multiple `WebSslProtocol` options with `-bor`,
 multiple protocols is not supported on all platforms.
 
 > [!NOTE]
-> On non-Windows platforms it may not be possible to supply `'Tls, Tls12'` as an option.
+> On non-Windows platforms it may not be possible to supply `'Tls, Tls12'` as an option. Support for
+> `'Tls13'` is not available on all Operating Systems and will need to be verified on a per
+> Operating System basis.
```
Comment (DCtheGeek) [inline reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md:987]: Don't think 'operating system' needs to be cased here or the line above.

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md [hunk 2]
```diff
@@ -982,15 +982,17 @@ using flag notation or combining multiple `WebSslProtocol` options with `-bor`,
 multiple protocols is not supported on all platforms.
 
 > [!NOTE]
-> On non-Windows platforms it may not be possible to supply `'Tls, Tls12'` as an option.
+> On non-Windows platforms it may not be possible to supply `'Tls, Tls12'` as an option. Support for
+> `'Tls13'` is not available on all Operating Systems and will need to be verified on a per
```
Comment (DCtheGeek) [inline reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md:986]: I know you are just keeping this format from the previous state, but is there a reason for the codefence and ticks together?

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md [hunk 3]
```diff
@@ -982,15 +982,17 @@ using flag notation or combining multiple `WebSslProtocol` options with `-bor`,
...
+> Operating System basis.
 
-This feature was added in PowerShell 6.0.0.
+This feature was added in PowerShell 6.0.0 and support for `'Tls13'` added in PowerShell 7.1.
```
Comment (DCtheGeek) [inline reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md:989]: To keep tense, probably "was added" in the new portion of the string.

**Review by DCtheGeek — CHANGES_REQUESTED** [review]

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md (hunk 2 above)
Comment (chasewilson) [reply reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md:986]: Actually no. I think code fence is going to benthe best option here.

Comment (chasewilson) [reply reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md]: fixed

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md (hunk 1 above)
Comment (chasewilson) [reply reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md]: fixed

**Review by DCtheGeek — APPROVED** [review]

**Pull request merged**